packages/dds/sequence/src/generateSequentialId.ts

```diff
@@ -0,0 +1,144 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const digitZero = "A";
```

Comment:
I think we should consider alternative representations and see which one is easier to work with.
For example, we can use actual digits and binary (base 256) representation(i.e. Uint8Array). I bet it will be easier to work with, with exception of allocation of buffer (as you need to know its size prior to allocation).

Reply:
Kept digits due to base 26 which gives more compactness